ontohgis.ttl

```diff
@@ -17236,6 +16854,21 @@ ontohgis:administrative_system_48 rdf:type owl:NamedIndividual ,
                                                                                  "Słowacja (1996-2020)"@pl .
 
 
+###  http://purl.org/ontohgis#administrative_system_49
+ontohgis:administrative_system_49 rdf:type owl:NamedIndividual ,
+                                           ontohgis:object_21 ;
+                                  ontohgis:endsDominationAt "1806-08-06"^^xsd:date ;
+                                  ontohgis:hasBeginning "800-12-25"^^xsd:date ;
+                                  ontohgis:hasEnd "1806-08-06"^^xsd:date ;
+                                  ontohgis:hasScope "terytorium wielokrotnie zmieniało swój zasięg, obejmowało głównie kraje Europy Środkowej i Wschodniej, w przeważającej części zamieszkiwane przez ludność posługującą się językiem niemieckim"@pl ;
+                                  ontohgis:individualHasExternalIdentifier 49 ;
+                                  ontohgis:startsDominationAt "962-02-02"^^xsd:date ;
```

Comment:
The value here should read "0962-02-02".

Reply:
Ok